OpenRA.Mods.Common/Traits/SupportPowers/NukePower.cs

```diff
@@ -108,7 +111,7 @@ public override void Activate(Actor self, Order order, SupportPowerManager manag
 				wsb.PlayCustomAnimation(self, info.ActivationSequence, () => wsb.CancelCustomAnimation(self));
 			}
 
-			var targetPosition = self.World.Map.CenterOfCell(order.TargetLocation);
+			var targetPosition = self.World.Map.CenterOfCell(order.TargetLocation) + new WVec(WDist.Zero, WDist.Zero, info.AirburstAltitude);
```

Comment:
WVec only has a WDist WDist WDist and an int int int constructor which matters in this case. Falling back to the int one would mean the WDist valuefrom the yaml would need to be converted and this is more readable IMO.